### ISSUE TITLE
Rename the "I installed via Office 365" checkbox to be "Enable Office 365/Click-to-Run Compatibility"

### ISSUE THREAD

**Issue by DrewNaylor** [DrewNaylor/UXL-Launcher#100]
# This issue described better here: https://github.com/DrewNaylor/UXL-Launcher/issues/100#issuecomment-395926431

**Comment by DrewNaylor** [comment]
These are the checkboxes on the General tab in the Options window. This was going to be replaced with one checkbox, but changing text is better.

**Comment by DrewNaylor** [comment]
Tooltip for "Use alternate 'Microsoft Office' folder":

"Recommended if using Office 2013. Instead of launching from 'Microsoft Office', we'll launch from 'Microsoft Office (version number)', where '(version number)' is the Office product version number. In Office 2013's case, this number is '15'.

**Comment by DrewNaylor** [comment]
Tooltip for "Enable Office 365/Click-to-Run Compatibility":

"Check this box if you're having trouble launching Office even with the rest of the options configured properly.

On some versions of Office such as Office 2016, Microsoft places the EXE files in a slightly different location from older versions of Office. This is due to a technology called 'Click-to-Run', or 'C2R' for short. This option uses that folder instead of the default."

**Comment by DrewNaylor** [comment]
This will require new code to be written for the "Use alternate 'Microsoft Office' folder" option. Maybe something like:

```vbnet
If My.Settings.alternateOfficeFolder = True Then
    stringAlternateOfficeFolder = My.Settings.userOfficeVersion
ElseIf My.Settings.alternateOfficeFolder = False Then
    stringAlternateOfficeFolder = ""
EndIf
```

This string would then be placed between "\Microsoft Office" and the "\" then the "userHasOfficeThreeSixFive" string kinda like so:

```vbnet
stringOfficeStringExample = "\Microsoft Office" & stringAlternateOfficeFolder & "\" & userHasOfficeThreeSixFiveString
```

This code isn't close to accurate, but it's just something I want to write down before I forget.

**Comment by DrewNaylor** [comment]
The "Enable Office 365/Click-to-Run Compatibility" checkbox should work just fine with the existing code, except that some code will need to be rewritten. For example, disabling one checkbox while certain Office versions are selected in the dropdown should no longer be required.

Also, `If My.Settings.userHasOfficeThreeSixFive = False` will replace the currently existing `If My.Settings.MSIInstallMethod = True` setting in currently existing code. These strings may not match exactly with the actual My.Settings strings in UXL Launcher, but they should be close. Other strings using `My.Settings.MSIInstallMethod` can be removed from the code since they would no longer be needed.

**Comment by DrewNaylor** [comment]
Actually, only rename the Office 365 checkbox and change its functionality as mentioned in https://github.com/DrewNaylor/UXL-Launcher/issues/100#issuecomment-362838841:

(Tooltip for "Enable Office 365/Click-to-Run Compatibility":

"Check this box if you're having trouble launching Office even with the rest of the options configured properly.

On some versions of Office such as Office 2016, Microsoft places the EXE files in a slightly different location from older versions of Office. This is due to a technology called 'Click-to-Run', or 'C2R' for short. This option uses that folder instead of the default.")

and https://github.com/DrewNaylor/UXL-Launcher/issues/100#issuecomment-362840736:

(The "Enable Office 365/Click-to-Run Compatibility" checkbox should work just fine with the existing code, except that some code will need to be rewritten. For example, disabling one checkbox while certain Office versions are selected in the dropdown should no longer be required.

Also, If My.Settings.userHasOfficeThreeSixFive = False will replace the currently existing If My.Settings.MSIInstallMethod = True setting in currently existing code. These strings may not match exactly with the actual My.Settings strings in UXL Launcher, but they should be close. Other strings using My.Settings.MSIInstallMethod can be removed from the code since they would no longer be needed.)

**Comment by DrewNaylor** [comment]
For those who have clicked the link in the Options window and are reading this, I would like to state that the text on the label doesn't look the best due to the Windows Forms UseCompatibleTextRendering property being set to True. I can't turn that off because that's the only way to use LinkLabel LinkArea.

Second, I need to explain the reason for this linklabel. In version 3.2, the top checkbox ("I installed via Office 365" in version 3.1) will be renamed to "Enable Office 365/Click-to-Run Compatibility" and will combine the functions of both the top checkbox and the bottom checkbox ("I installed via MSI").
